libbeat/outputs/kafka/kafka.go

```diff
@@ -225,6 +250,7 @@ func newKafkaConfig(config *kafkaConfig) (*sarama.Config, error) {
 		retryMax = 1000
 	}
 	k.Producer.Retry.Max = retryMax
+	// TODO: k.Producer.Retry.Backoff = ?
```

Comment:
yes. It's some samara internal config with default 100ms kinda overlapping with libbeat functionality (not critical, though). Added `TODO` so I will reconsider a parameter maybe another time instead of hiding the fact something might be "incomplete".